configuration/secrets.rst

```diff
@@ -0,0 +1,321 @@
+                'password' => '%env(secret:DATABASE_PASSWORD)%',
+            ]
+        ]);
+
+The actual value will be resolved at runtime: container compilation and cache
+warmup don't need the **decryption key**.
+
+List Existing Secrets
+---------------------
+
+Everybody is allowed to list the secrets names with the command
+``secrets:list``. If you have the **decryption key** you can also reveal the
+secrets' values by passing the ``--reveal`` option:
+
+.. code-block:: terminal
+
+    $ php bin/console secrets:list --reveal
+
+     ------------------- ------------ -------------
+      Name                Value        Local Value
+     ------------------- ------------ -------------
+      DATABASE_PASSWORD   "my secret"
+     ------------------- ------------ -------------
+
+Remove Secrets
+--------------
+
+Symfony provides a convenient command to remove a Secret:
+
+.. code-block:: terminal
+
+    $ php bin/console secrets:remove DATABASE_PASSWORD
+
+Local secrets: Overriding Secrets Locally
+-----------------------------------------
+
+The ``dev`` environment secrets should contain nice default values for development.
+But sometimes a developer *still* needs to override a secret value locally when
+developing.
+
+Most of the ``secrets`` commands - including ``secrets:set`` - have a ``--local``
+option that stores the "secret" in the ``.env.{env}.local`` file as a standard
+environment variable. To override the ``DATABASE_PASSWORD`` secret locally, run:
+
+.. code-block:: terminal
+
+    $ php bin/console secrets:set DATABASE_PASSWORD --local
+
+If you entered ``root``, you will now see this in your ``.env.dev.local`` file:
+
+.. code-block:: bash
+
+    DATABASE_PASSWORD=root
+
+This will *override* the ``DATABASE_PASSWORD`` secret because environment variables
+always "win" over secrets.
+
+Listing the secrets will now also display the local variable:
+
+.. code-block:: terminal
+
+    $ php bin/console secrets:list --reveal
+     ------------------- ------------- -------------
+      Name                Value         Local Value
+     ------------------- ------------- -------------
+      DATABASE_PASSWORD   "dev value"   "root"
+     ------------------- ------------- -------------
+
+Symfony also provides the ``secrets:decrypt-to-local`` command to decrypts
+all secrets and stores them in the local vault and ``secrets:encrypt-from-local``
+to encrypt all local secrets to the vault.
+
+Secrets in the test Environment
+-------------------------------
+
+If you add a secret in the ``dev`` and ``prod`` environments, it will be missing
+from the ``test`` environment. You *could* create a "vault" for the ``test``
+environment and define the secrets there. But an easier way is to set the test
+values via the ``.env.test`` file:
+
+.. code-block:: bash
+
+    # .env.test
+    DATABASE_PASSWORD="testing"
+
+Deploy secrets to production
+----------------------------
+
+Because the **decryption key** is not committed, you'll need to manually
```

Comment:
Maybe reword, as a reminder

Due to the fact that decryption keys should never be commited, you will need ...

Wdyt?